executor/api/rest/server.go

```diff
@@ -344,3 +346,30 @@ func (r *SeldonRestApi) predictions(w http.ResponseWriter, req *http.Request) {
 	}
 	r.respondWithSuccess(w, http.StatusOK, resPayload)
 }
+
+func (r *SeldonRestApi) graph_metadata(w http.ResponseWriter, req *http.Request) {
```

Comment:
Why is it underscore instad of camelcase? Probably best to stick with camelcase for consistency

Reply:
Will change, just muscle memory from Python :)